src/main/java/net/sf/jabref/logic/xmp/XMPUtil.java

```diff
@@ -832,7 +832,7 @@ private static void writeToDCSchema(XMPSchemaDublinCore dcSchema,
              *
              * Description: Relationships to other documents.
              *
-             * Bibtex-Fields used: bibtextype
+             * Bibtex-Fields used: entrytype
              */
```

Comment:
Probably. But to be honest I didn't understand the code more than I'm quite sure that I should change the comment.